### PR TITLE
update layout, slim, bootstrap 4

### PR DESCRIPTION
**Pull request by FeshchenkoMaksym**
update layout using slim;
setup bootstrap 4;
add design to the list of posts:

### CODE REVIEW

**Comment by FeshchenkoMaksym** [comment]
screenshot

**Comment by FeshchenkoMaksym** [comment]
scren
visual update

File: app/assets/stylesheets/layout/header.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  height: 400px;
+  .container-fluid-width {
+    background-color: $light_gray; //rgba(242, 242, 242, 0.6)
+    border-bottom: 1px solid black;
+  }
+  .title {
+    display: flex;
+    justify-content: center;
+    font-size: xx-large;
+    margin-top: 1.4em;
```
Comment (Vitaliy-Maznyi) [inline app/assets/stylesheets/layout/header.scss]: why do you need this margin-top here?

File: app/assets/stylesheets/shared/sizes.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$padding_topic_posts: 8px 16px 8px 16px;
```
Comment (Vitaliy-Maznyi) [inline app/assets/stylesheets/shared/sizes.scss]: you don't need to save all of the sizes inside the variables. Set this padding directly in the class where you've used this variable and remove this variable and the variable below

**Review by Vitaliy-Maznyi — CHANGES_REQUESTED** [review]
almost done. Remove the sizes.scss file, you don't need it for now, I'll explain you later where and how it should be used.

**Review by Vitaliy-Maznyi — APPROVED** [review]
let's merge

**Pull request merged**